spring-cloud-stream-schema/pom.xml

```diff
@@ -39,6 +39,10 @@
 			<version>${avro.version}</version>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+		</dependency>
```

Comment:
@5aab We don't generally prefer the use of Lombok in framework code. I realize that you are only using this for tests, but this could introduce potential IDE issues for other users. Better to remove it and use a standard POJO for the tests.